tests/pytests/test_replicate.py

```diff
@@ -115,7 +115,7 @@ def testDropReplicate():
   master.execute_command('FT.DROPINDEX', 'idx', 'DD')
 
   # check that same docs were deleted by master and slave
-  time.sleep(0.1)
+  time.sleep(0.5)
```

Comment:
@ashtul we need to find a deterministic way to test it. If you increase the time to much it might be that the indexing is done on both primary and replica and then you check nothing ...

Reply:
This is a wait after we already dropped the index.
It gives enough time to delete the documents before calling `KEYS *`.

Reply:
ohh so why not do wait command which will wait until the master and the slave will finished the drop command:
https://redis.io/commands/wait

Reply:
This issue was moved to another PR